official/resnet/estimator_cifar_benchmark.py

```diff
@@ -29,22 +29,33 @@
 from official.resnet import cifar10_main as cifar_main
 from official.utils.logs import hooks
 
-DATA_DIR = '/data/cifar10_data/cifar-10-batches-bin'
-
 
 class EstimatorCifar10BenchmarkTests(tf.test.Benchmark):
   """Benchmarks and accuracy tests for Estimator ResNet56."""
 
   local_flags = None
 
-  def __init__(self, output_dir=None):
+  def __init__(self, output_dir=None, root_data_dir=None):
+    """A benchmark class.
+
+    Args:
+      output_dir: directory where to output e.g. log files
+      root_data_dir: directory under which to look for dataset
+    """
+
     self.output_dir = output_dir
 
+    if root_data_dir is None:
+      self.data_dir = '/data/cifar10_data/cifar-10-batches-bin'
+    else:
+      self.data_dir = os.path.join(root_data_dir,
+                                   'cifar10_data/cifar-10-batches-bin')
```

Comment:
This path should not have /cifar-10-batches-bin in it that greatly restricts the structure the user would setup.  We can update the jobs that download CIFAR-10 data to put it directly in cifar10_data.  Long story, we had to tweak the underlying code to resolve this issue internally due to how we stored the data in CNS and realized we should not go more than one folder deep as far as assumptions of names unless the structure is necessary and well accepted as the "only" way to setup the data.

Reply:
Very good point. It is fixed now.